styx-scheduler-service/src/main/java/com/spotify/styx/monitoring/Stats.java

```diff
@@ -57,5 +57,9 @@
 
   void exitCodeMismatch();
 
+  void registerResourceConfigured(String resource, Gauge<Long> resourceCount);
+
+  void resourceUsed(String resource, long used);
```

Comment:
I guess it's inconsistent to have the other method start with "register"; either all of them should or none of them should.